lib/apollo-federation/schema.rb

```diff
@@ -64,7 +71,7 @@ def federation_2_prefix
 
         <<~SCHEMA
           extend schema
-            @link(url: "https://specs.apollo.dev/federation/v2.3"#{federation_namespace}, import: [#{(IMPORTED_DIRECTIVES.map { |directive| "\"@#{directive}\"" }).join(', ')}])
+            @link(url: "https://specs.apollo.dev/federation/v2.3"#{federation_namespace}, import: [#{(@imported_directives.map { |directive| "\"@#{directive}\"" }).join(', ')}])
```

Comment:
```suggestion
            @link(url: "https://specs.apollo.dev/federation/v2.3"#{federation_namespace}, import: [#{(get_imported_directives.map { |directive| "\"@#{directive}\"" }).join(', ')}])
```

Reply:
I was wondering which one to call here. I chose wrong 😅